### ISSUE TITLE
Add to the vocabulary a category "other usages/unspecified” 

### ISSUE THREAD

**Issue by giuliamarangoni** [lrosenthol/opt-out-vocab#8]
In order to be future-proof, the vocabulary should include a category "other uses - unspecified", which falls under TDM uses as defined by art 4 of DSM Directive. As discussed in Brussels, we are all aware that there may be usages other than machine learning or Gen AI training that are not yet defined (they will be in due course). It should be clear that such uses fall within the scope of the TDM opt-out as defined by art.4 of DSM Directive

**Comment by paul2keller** [comment]
I don’t think this is either practical or desirable. What we discussed in brussels is that the vocabulary should be expandable to account for use cases that emerge in the future. 

But to be practicable such used cases must be precisely defined (and "other usages-unspecified” is the opposite of that). I would propose that we will note that the vocabulary can be expanded as additional uses cases emerge to account for the scenario that you describe, but that we only add additional uses cases if there is a stable definition that is shared among stakeholders.

**Comment by lrosenthol** [comment]
Agree with @paul2keller - I'll add something talking about extensibility and that it can be expanded.   This will also fit nicely into the general TDM category.

**Comment by paul2keller** [comment]
I think we can close this issue @lrosenthol
